ui/app/templates/components/console/command-input.hbs

```diff
@@ -12,17 +12,14 @@
   <input onkeyup={{action "handleKeyUp"}} value={{this.value}} autocomplete="off" spellcheck="false" />
   <ToolTip @horizontalPosition="auto-right" @verticalPosition="above" as |d|>
     <d.Trigger>
-      <button
-        type="button"
-        class={{concat "button is-compact" (if this.isFullscreen " active")}}
+      <Hds::Button
+        class="hds-side-nav__icon-button"
         {{on "click" (action "fullscreen")}}
         data-test-tool-tip-trigger
-      >
-        <Icon
-          @name={{if this.isFullscreen "minimize" "maximize"}}
-          aria-label={{if this.isFullscreen "Minimize" "Maximize"}}
-        />
-      </button>
+        @icon={{if this.isFullscreen "minimize" "maximize"}}
+        @text={{if this.isFullscreen "Minimize" "Maximize"}}
+        @isIconOnly={{true}}
```

Comment:
Should `isIconOnly` be set if there is text?

Reply:
Good question! `@text` is a required arg for all HDS buttons. You're right below, it's used for the `aria-label` when `isIconOnly` is true